lib/widgets/custom_app_bar.dart

```diff
@@ -8,11 +9,24 @@ class CustomAppBar extends ConsumerWidget with PreferredSizeWidget {
   const CustomAppBar({
     Key? key,
     required this.title,
-    required this.image,
   }) : super(key: key);
-
   final String title;
-  final Image image;
+  Widget _buildImageChild() {
+    if (title == "Committees & Events") {
+      return Image.asset(
+        ImageAssets.committes,
+        width: 177,
+      );
+    } else if (title == "Training & Placement Cell") {
+      return Image.asset(
+        ImageAssets.tpo,
+        width: 177,
+      );
+    } else if (title == "") {
+      return Image.asset(ImageAssets.committes, width: 177);
+    }
+    return const SizedBox(width: 177);
+  }
```

Comment:
Remove this, previous implementation felt better

Reply:
Yeah but in this way, the widget can be customised as per app bar. We can do it for weird ones like Login also

Reply:
Anyways will fix this